pkg/controller/persistentvolume/persistentvolume_provisioner_controller.go

```diff
@@ -345,6 +347,11 @@ func newProvisioner(plugin volume.ProvisionableVolumePlugin, claim *api.Persiste
 		AccessModes:                   claim.Spec.AccessModes,
 		PersistentVolumeReclaimPolicy: api.PersistentVolumeReclaimDelete,
 		CloudTags:                     &tags,
+		ClusterName:                   controller.clusterName,
+	}
+
+	if pv != nil {
+		volumeOptions.PVName = pv.Name
```

Comment:
@saad-ali strictly speaking, we are assigning PV.Name from Kube to GCEPD.Name in the provider.  The name field exists on both, so we think it would be beneficial for them to match.